components/brave_wallet_ui/utils/network-utils.ts

```diff
@@ -39,3 +39,30 @@ export const reduceNetworkDisplayName = (name: string) => {
     }
   }
 }
+
+export const GetNetworksByCoinType = (networks: BraveWallet.NetworkInfo[], coin: BraveWallet.CoinType): BraveWallet.NetworkInfo[] => {
```

Comment:
Why are these function names in Pascal case?

Reply:
Good call! Changed to `camelCase`